add a SECURITY notice (#147)
lidofinance/core: SECURITY.md

# Security Policy

## Reporting a Vulnerability

> The Lido team appreciates your efforts to responsibly disclose your findings and will make every effort to acknowledge your contributions.

Reviewer: No 'Lido team' please :)

Lido contributors :pray:

To report a security issue, please use the [Lido Bug Bounty program on Immunefi](https://immunefi.com/bounty/lido).

The Immunefi platform allows the Lido team to better track and respond to vulnerabilities, while also providing an opportunity for researchers to be rewarded for their findings.

> Please DO NOT file a public issue on GitHub or disclose the vulnerability publicly in any way before it has been addressed by the Lido core contributors.

Reviewer 2: ```suggestion
> [!IMPORTANT]
> Please DO NOT file a public issue on GitHub or disclose the vulnerability publicly in any way before it has been addressed by the Lido core contributors.
```

## Scope

For details on what is considered in scope for the bug bounty program, please refer to the Lido Bug Bounty page on Immunefi.

## Process

1. Submit your report through the Immunefi platform.

> 2. The Lido team will assess the report and may ask for additional information or clarification.

Reviewer 2: ```suggestion
2. The Lido contributors will evaluate your report and may request additional information or clarification.
```

3. Once validated, the team will work on a fix and coordinate the release process.

> 4. After the fix has been deployed, the team will publish a security advisory and acknowledge your contribution (if you wish to be credited).

Reviewer 2: ```suggestion
4. After the fix is deployed, a security advisory will be published, and your contribution will be acknowledged (if you wish to be credited).
```
